app/helpers/application_helper.rb

```diff
@@ -1,2 +1,16 @@
 module ApplicationHelper
+  def area_tag(area, classes = [])
+    x, y = area.coordinate&.slice(:x, :y)&.values_at(:x, :y) || [0, 0]
+    classes.push('area', area.occupied ? 'occupied' : 'free')
+    content_tag(:div,
+                nil,
+                'title' => area.name,
```

Comment:
Rocket notation is sort of deprecated in newer ruby version. Go with symbol keys instead `title: area.name`

Reply:
What about hyphenated keys like `data-type`? For now I've done `:'data-type' => area.type,` but that's pretty ugly.

Reply:
Oh right i missed those 😅 

![image](http://www.banklawyersblog.com/.a/6a00d8341c652b53ef015391c79429970b-800wi)